ext/node/polyfills/_http_outgoing.ts

```diff
@@ -108,6 +108,8 @@ export function OutgoingMessage() {
   this._onPendingData = nop;
 
   this._bodyWriter = null;
+
+  this._listenerSet = false;
```

Comment:
Maybe use a private field here (`#listenerSet`) as this field will be visible to the users?

Reply:
Ah, it looks like this field isn't used anywhere 😓  Removing